doc/manipulating_images/input_output.rst

```diff
@@ -234,6 +234,16 @@ data, which we call Niimgs or Niimg-4D. Accepted input arguments are:
    If you provide a sequence of Nifti images, all of them must have the same
    affine.
 
+.. topic:: Decreasing memory used when loading Nifti images
+
+   When Nifti images are stored compressed (.nii.gz), loading them directly
+   consumes more memory. As a result, large 4D images may
+   raise "MemoryError", especially on smaller computers and when using Nilearn
+   routines that require intensive 4D matrix operations. One step to improve
+   the situation may be to gunzip the data onto disk as an initial step.
```

Comment:
Since there are multiple software to achieve this outcome, I might be more general:

```suggestion
   the situation may be to decompress the data onto disk as an initial step.
```

Reply:
I agree